src/routes/backoffice.ts

```diff
@@ -235,6 +235,7 @@ export const setIdentification = async (req, res) => {
   res.status(204).send();
 };
 
+// sets identification state from the panel
```

Comment:
So by default the user identification is set to `green` : https://github.com/kontist/mock-solaris/pull/66/files#diff-7491a437e45cd55643d2423c1c037616158480686df6082b2e6ba7645dd872a3R204-R213
And then we can manually change these screening/risk/vetting values for testing. 

I'm just wondering about the `IdentificationStatus`, if it would need to be automatically changed accordingly to the value we chose for the screening/risk/vetting. But maybe that's not needed for our mockSolaris, we should keep it simple. 
Which means that on the BE side we need to validate carefully each value to be sure that we can trigger or not the account creation.